sdk/eventhub/Azure.Messaging.EventHubs.Experimental/src/Azure.Messaging.EventHubs.Experimental.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>Experimental types that are being evaluated and which may or may not eventually be promoted to official status in the Event Hubs client libraries.  For more information about Event Hubs, see https://azure.microsoft.com/en-us/services/event-hubs/</Description>
```

Comment:
I stole the format and high-level content from `Azure.Core.Experimental` here and in the markdown assets.